### ISSUE TITLE
`Microsoft.Network/virtualHubs`: `propagateStaticRoutes` of `hubVirtualNetworkConnections` is marked as `readonly` but actually editable

### ISSUE THREAD

**Issue by ziyeqf** [Azure/azure-rest-api-specs#23920]
The property is marked as `read-only` but could be edit on Portal. https://github.com/Azure/azure-rest-api-specs/blob/fb9c8e2ca33e9723c2b2fc849f627329067feb54/specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/virtualWan.json#L9201

**Comment by tombuildsstuff** [comment]
@ziyeqf which API version is the Portal using, out of interest?

**Comment by tombuildsstuff** [comment]
FWIW this property is also `readonly` in that API Version, so it'd be good to confirm the Request/Response payloads being sent, to confirm that this is a Swagger bug (and that it's not using a different API to do the update, for example?)

![Screenshot 2023-05-30 at 12 22 13](https://github.com/Azure/azure-rest-api-specs/assets/666005/efd6ac45-5992-482d-811e-03178bac7d45)

**Comment by ziyeqf** [comment]
Yes I can confirm it's editable on business (on Portal), and in request it's using the same property.
![image](https://github.com/Azure/azure-rest-api-specs/assets/51212351/eccdaf3f-9838-4513-be40-e7280ab705aa)
![image](https://github.com/Azure/azure-rest-api-specs/assets/51212351/72e6a9c8-5e95-4e02-844d-151255b531b2)

And just confirmed the Portal is also using `2022-01-01`, the previous answer was mistake..

**Comment by ziyeqf** [comment]
> @ziyeqf fwiw from what I can see that API version doesn't contain this field - so that looks to be another issue here?

okay, here you go https://github.com/Azure/azure-rest-api-specs/issues/24246
